zerver/tests/test_bugdown.py

```diff
@@ -530,13 +530,12 @@ def make_inline_twitter_preview(url: Text, tweet_html: Text, image_html: Text=''
                'http://twitter.com/wdaher/status/287977969287315457 '
                'http://twitter.com/wdaher/status/287977969287315457')
         converted = bugdown_convert(msg)
-        self.assertEqual(converted, '<p>%s %s %s %s</p>\n%s%s%s' % (
+        self.assertEqual(converted, '<p>%s %s %s %s</p>\n%s%s' % (
             make_link('http://twitter.com/wdaher/status/287977969287315456'),
             make_link('http://twitter.com/wdaher/status/287977969287315457'),
             make_link('http://twitter.com/wdaher/status/287977969287315457'),
             make_link('http://twitter.com/wdaher/status/287977969287315457'),
             make_inline_twitter_preview('http://twitter.com/wdaher/status/287977969287315456', normal_tweet_html),
-            make_inline_twitter_preview('http://twitter.com/wdaher/status/287977969287315457', normal_tweet_html),
             make_inline_twitter_preview('http://twitter.com/wdaher/status/287977969287315457', normal_tweet_html)))
```

Comment:
We still want a test for 4 distinct URLs, so please add a new test covering that situation.